docs/reference/hip_runtime_api/modules_list.rst

```diff
@@ -0,0 +1,28 @@
+* :ref:`driver_reference`
+* :ref:`device_management_reference`
+* :ref:`execution_control_reference`
+* :ref:`error_handling_reference`
+* :ref:`stream_management_reference`
+* :ref:`stream_memory_operations_reference`
+* :ref:`event_management_reference`
+* :ref:`memory_management_reference`
+
+  * :ref:`external_resource_interoperability_reference`
+  * :ref:`stream_ordered_memory_allocator_reference`
+  * :ref:`unified_memory_reference`
+  * :ref:`virtual_memory_reference`
+  * :ref:`texture_management_reference`
+  * :ref:`texture_management_deprecated_reference`
```

Comment:
Do we need to list the deprecated modules here? They will also appear in the Deprecated API list. If the intention with deprecation is to get people to stop using the features, then listing them once under Deprecated APIs should be enough.

Reply:
We need to keep it to be able to link from deprecated list.